### ISSUE TITLE
reproduce the exp results

### ISSUE THREAD

**Issue by fAnchao1** [JCruan519/VM-UNet#47]
i see that there is a seed fixed in the train.py. but i can not get the consistent results every training.

**Comment by JAYCHOU2020** [comment]
> i see that there is a seed fixed in the train.py. but i can not get the consistent results every training.



> i see that there is a seed fixed in the train.py. but i can not get the consistent results every training.

Do you get 81.3 miou after isic18 training?

**Comment by fAnchao1** [comment]
yes. but i use the depth [2,2,9,2] and vmamba_small_e238_ema.pth pretrained model. 
but anthor claim that 81.3 is get using the depths [2,2,2,2],which i can not get the results.

**Comment by JAYCHOU2020** [comment]
> vmamba_small_e238_ema.p

thanks!!!!!!  I'll try it

**Comment by JAYCHOU2020** [comment]
> yes. but i use the depth [2,2,9,2] and vmamba_small_e238_ema.pth pretrained model. but anthor claim that 81.3 is get using the depths [2,2,2,2],which i can not get the results.

thanks!!!!!!  I'll try it

**Comment by sakurarma** [comment]
@fAnchao1 I meet the same problem. I used the same parameters and fixed seed on another task, but the results were completely different each time. Have you solved it?

**Comment by fAnchao1** [comment]
@sakurarma not yet

**Comment by 1362356431** [comment]
I meet the same problem.I just changed the dataset，but the results were diffident each time on the same on the same GPU.

**Comment by 1362356431** [comment]
I meet the same problem.I just changed the dataset，but the results were diffident each time on the same GPU.

**Comment by JAYCHOU2020** [comment]
> yes. but i use the depth [2,2,9,2] and vmamba_small_e238_ema.pth pretrained model. but anthor claim that 81.3 is get using the depths [2,2,2,2],which i can not get the results.

i still cant get the 81.3 miou ,What is your batch size setting? May i see your train.info.log？thanks a lot!

**Comment by fAnchao1** [comment]
32，what&nbsp;miou&nbsp;you&nbsp;get


---Original---
From: ***@***.***&gt;
Date: Sat, May 4, 2024 13:53 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [JCruan519/VM-UNet] reproduce the exp results (Issue #47)




  
yes. but i use the depth [2,2,9,2] and vmamba_small_e238_ema.pth pretrained model. but anthor claim that 81.3 is get using the depths [2,2,2,2],which i can not get the results.
  
i still cant get the 81.3 miou ,What is your batch size setting? May i see your train.info.log？thanks a lot!
 
&mdash;

**Comment by JAYCHOU2020** [comment]
> 32，what&nbsp;miou&nbsp;you&nbsp;get
> […](#)
> ---Original--- From: ***@***.***&gt; Date: Sat, May 4, 2024 13:53 PM To: ***@***.***&gt;; Cc: ***@***.******@***.***&gt;; Subject: Re: [JCruan519/VM-UNet] reproduce the exp results (Issue #47) yes. but i use the depth [2,2,9,2] and vmamba_small_e238_ema.pth pretrained model. but anthor claim that 81.3 is get using the depths [2,2,2,2],which i can not get the results. i still cant get the 81.3 miou ,What is your batch size setting? May i see your train.info.log？thanks a lot! &mdash; Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

only get miou: 0.8029449451678118
